### PR TITLE
Show Tags on BoxView

### PR DESCRIPTION
**Pull request by vahidbazzaz**
In this PR, changes are made to the BoxView component to display tags.

### CODE REVIEW

**Comment by vahidbazzaz** [comment]
[2.0 BoxView: tags are not showing on the BoxView](https://trello.com/c/zHbKBzmI)

File: react/src/views/Box/components/BoxDetails.tsx [hunk 1]
```diff
@@ -1,312 +1,306 @@
...
-import {
-  BoxByLabelIdentifierQuery,
-  UpdateLocationOfBoxMutation,
-} from "types/generated/graphql";
+import { BoxByLabelIdentifierQuery, UpdateLocationOfBoxMutation } from "types/generated/graphql";
 import { useGetUrlForResourceHelpers } from "utils/hooks";
 import { distroEventStateHumanReadableLabels } from "views/Distributions/baseData";
 import DistributionEventTimeRangeDisplay from "views/Distributions/components/DistributionEventTimeRangeDisplay";
+import { Tag, TagLabel, TagLeftIcon } from "@chakra-ui/react";
+import { Style } from "victory";
```
Comment (HaGuesto) [inline react/src/views/Box/components/BoxDetails.tsx:22]: what is this for?

File: react/src/views/Box/components/BoxDetails.tsx [hunk 2]
```diff
@@ -1,312 +1,306 @@
...
-      color = "#EB404A";
-    } else {
-      color = "#0CA789";
-    }
-    return color;
-  };
+    const statusColor = (value) => {
+        let color;
+        if (value === "Lost" || value === "Scrap") {
+            color = "#EB404A";
```
Comment (HaGuesto) [inline react/src/views/Box/components/BoxDetails.tsx:48]: are these colors not defined somewhere centrally?

File: react/src/views/Box/components/BoxDetails.tsx [hunk 3]
```diff
@@ -1,312 +1,306 @@
...
-            </Heading>
-            <Flex>
-              <Text>
-                <b>State:&nbsp;</b>
-              </Text>
-              <Text color={statusColor(boxData.state)}><b>{boxData.state}</b></Text>
-            </Flex>
-          </Flex>
+    if (boxData == null) {
+        console.error("BoxDetails Component: boxData is null");
```
Comment (HaGuesto) [inline react/src/views/Box/components/BoxDetails.tsx:58]: maybe add a comment: "TODO: change error handling"

Comment (vahidbazzaz) [reply react/src/views/Box/components/BoxDetails.tsx:58]: @HaGuesto I think for this we also have task but that totally make sense to put a note

**Review by HaGuesto — COMMENTED** [review]
hard to code review due to the formatting changes

**Pull request merged**